examples/notebooks/change-settings.ipynb

```diff
@@ -1,11 +1,446 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Changing settings when solving PyBaMM models\n",
+    "\n",
+    "[This](./models/SPM.ipynb) example notebook showed how to run an SPM battery model, using the default parameters, discretisation and solvers that were defined for that particular model. Naturally we would like the ability to alter these options on a case by case basis, and this notebook gives an example of how to do this, again using the SPM model.\n",
+    "\n",
+    "### Table of Contents\n",
+    "1. [Default SPM model](#default)\n",
+    "1. [Changing the parameters](#parameters)\n",
+    "1. [Changing the discretisation](#discretisation)\n",
+    "1. [Changing the solver](#solver)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The default SPM model  <a name=\"default\"></a>\n",
+    "\n",
+    "Below is the code to define and run the default SPM model included in PyBaMM (if this is unfamiliar to you, please see [this](./models/SPM.ipynb) notebook for more details)"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "import pybamm\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "os.chdir(pybamm.__path__[0]+'/..')\n",
+    "\n",
+    "# create the model\n",
+    "model = pybamm.lithium_ion.SPM()\n",
+    "\n",
+    "# set the default model geometry\n",
+    "geometry = model.default_geometry\n",
+    "\n",
+    "# set the default model parameters\n",
+    "param = model.default_parameter_values\n",
+    "\n",
+    "# set the parameters for the model and the geometry\n",
+    "param.process_model(model)\n",
+    "param.process_geometry(geometry)\n",
+    "\n",
+    "# mesh the domains\n",
+    "mesh = pybamm.Mesh(geometry, model.default_submesh_types, model.default_var_pts)\n",
+    "\n",
+    "# discretise the model equations\n",
+    "disc = pybamm.Discretisation(mesh, model.default_spatial_methods)\n",
+    "disc.process_model(model)\n",
+    "\n",
+    "# Solve the model at the given time points\n",
+    "solver = model.default_solver\n",
+    "n = 100\n",
+    "t_eval = np.linspace(0, 2, n)\n",
```

Comment:
using something like `t_eval = np.linspace(0, 0.2, n)` will give smoother curves (because we hit terminal voltage / concentration at t=0.2)